platforms/jvm/scala/src/main/java/org/gradle/api/plugins/scala/ScalaPluginExtension.java

```diff
@@ -36,4 +39,16 @@ public interface ScalaPluginExtension {
      * @since 6.0
      */
     Property<String> getZincVersion();
+
+    /**
+     * The version of the Scala to use for compiling Scala code.
+     *
+     * @return The scala version
+     *
+     * @since 8.12
+     */
+    @Optional
+    @Incubating
+    Property<String> getScalaVersion();
```

Comment:
💭 As this property already exists within an extension named `scala`, the prefix seems redundant.  I think this would be better as just `version`.

Reply:
I thought the same thing, but since there is a zincVersion right next to it I felt specifying scala for this version reduced ambiguity

Reply:
Ah, ok.  Probably worth keeping then.